cmd/root.go

```diff
@@ -55,6 +56,33 @@ func RunCommand() *cobra.Command {
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 			output.SetOut(cmd.OutOrStdout())
 			output.SetErr(cmd.ErrOrStderr())
+
+			// Load config from the file (or default if it doesn't exist)
+			configHome, err := getConfigHome(opts)
+			if err != nil {
+				output.Errorf("Failed to read base config directory")
+				output.Infof("Use the --config flag or set the $%s environment variable to provide a default", constants.KitopsHomeEnvVar)
+				output.Debugf("Error: %s", err)
+				return errors.New("exit")
+			}
+
+			configPath := filepath.Join(configHome, constants.ConfigFileName)
```

Comment:
We also probably want to avoid `filepath.Join` here, moving it instead to a fucntion -- see e.g. [`constants.StoragePath()`](https://github.com/jozu-ai/kitops/blob/6469d0840e440b7d841a4abe189cfddd8f6ff40b/pkg/lib/constants/consts.go#L103-L105)

This would allow us to modify the config path more easily in the future.

Reply:
Refactored!